src/main/java/org/jitsi/videobridge/octo/OctoRelayImpl.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright @ 2015-2017 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.videobridge.octo;
+
+import org.ice4j.socket.*;
+import org.jitsi.util.*;
+
+import java.net.*;
+
+/**
+ * Implements a bridge-to-bridge (Octo) relay.
+ *
+ * @author Boris Grozev
+ */
+public class OctoRelayImpl
```

Comment:
should this be named with 'Impl' if it's not an implementation of any interface?  just a question, not sure what our convention is.